plugins/octant/cmd/antrea-octant-plugin/main.go

```diff
@@ -42,28 +39,36 @@ const (
 	overviewTitle   = "Overview"
 	controllerTitle = "Antrea Controller Info"
 	agentTitle      = "Antrea Agent Info"
-	versionCol      = "Version"
-	podCol          = "Pod"
-	nodeCol         = "Node"
-	serviceCol      = "Service"
-	crdCol          = "Monitoring CRD"
-	subnetCol       = "NodeSubnet"
-	bridgeCol       = "OVS Bridge"
-	podNumCol       = "Local Pod Num"
-	heartbeatCol    = "Last Heartbeat Time"
+	traceflowTitle  = "Trace Flow"
 )
 
-func main() {
-	// Remove the prefix from the go logger since Octant will print logs with timestamps.
-	log.SetPrefix("")
+type octantPlugin struct {
+	client     *clientset.Clientset
+	graph      string
+	lastTfName string
+}
+
+func newOctantPlugin() *octantPlugin {
 	config, err := clientcmd.BuildConfigFromFlags("", os.Getenv(kubeConfig))
 	if err != nil {
 		log.Fatalf("Failed to build kubeConfig %v", err)
 	}
-	client, err = clientset.NewForConfig(config)
+	client, err := clientset.NewForConfig(config)
 	if err != nil {
-		log.Fatalf("Failed to create K8s client for antrea-octant-plugin %v", err)
+		log.Fatalf("Failed to create K8s client for %s: %v", pluginName, err)
+	}
+	return &octantPlugin{
+		client:     client,
+		graph:      "",
+		lastTfName: "",
 	}
+}
+
+func main() {
+	octantPlugin := newOctantPlugin()
+
+	// Remove the prefix from the go logger since Octant will print logs with timestamps.
+	log.SetPrefix("")
 
 	// This plugin is interested in AntreaControllerInfo and AntreaAgentInfo.
```

Comment:
I think we can remove these two definitions since we did not actually use SupportsPrinterConfig and SupportsTab.
Could you please verify this?

Reply:
Removed octantPlugin and log.SetPrefix("").

Reply:
Actually, I mean GVK definitions of AntreaControllerInfo and AntreaAgentInfo.